Resources/Prototypes/Recipes/Reactions/drinks.yml

```diff
@@ -878,3 +868,13 @@
       amount: 1
   products:
     TheMartinez: 6
+
+- type: reaction
+  id: WhiteGilgamesh
+  reactants:
+    Beer:
+      amount: 2
+    Milk:
+      amount: 1
+  products:
+    WhiteGilgamesh: 2
```

Comment:
I feel like 3 reagents becoming 2 violates some kind of convention on drinks.

Reply:
fuck i forgot that bit, i'll fix it